### ISSUE TITLE
"Remove all" link does not meet colour contrast accessibility when disabled

### ISSUE THREAD

**Issue by joshsadam** [phac-nml/irida-next#1000]
![Image](https://github.com/user-attachments/assets/7787d15b-669c-4f0a-bdaa-e20dba7e1383)

Colour contrast: 1.49
